blackbelt/tasks.py

```diff
@@ -15,12 +15,15 @@ def list_commands(self, ctx):
         return rv
 
     def get_command(self, ctx, name):
-        ns = {}
-        fn = os.path.join(plugin_folder, name + '.py')
-        with open(fn) as f:
-            code = compile(f.read(), fn, 'exec')
-            eval(code, ns, ns)
-        return ns['cli']
+        try:
+            ns = {}
+            fn = os.path.join(plugin_folder, name + '.py')
+            with open(fn) as f:
+                code = compile(f.read(), fn, 'exec')
+                eval(code, ns, ns)
+            return ns['cli']
+        except Exception:
+            pass
```

Comment:
I don't know exactly how the commands are being executed in the context of the whole application, but whouldn't this silence every exception in black-belt?

I'd either
- silence only exceptions being raised for not existing commands 
- make semi-silent all exceptions (would print only name of error or something) and then introduce `--debug` argument in case I'd like to debug and see the stacktrace